doc/api/api_changes/code_removal.rst

```diff
@@ -36,8 +36,15 @@ favor of the synonym ``"auto"``.
 The ``shading`` kwarg to ``pcolor`` has been removed.  Set ``edgecolors``
 appropriately instead.
 
+
 Removed internal functions
 --------------------------
 
 The ``matplotlib.backends.backend_ps.seq_allequal`` function has been removed.
 Use ``np.array_equal`` instead.
+
+The deprecated `matplotlib.rcsetup.validate_maskedarray`,
```

Comment:
Err, actually, since these are now deleted, they should be code, not references.